portal/models.py

```diff
@@ -37,6 +37,14 @@
     ('other', 'Other'),
 )
 
+OS_TYPES = (
+    ('linux', 'Linux'),
+    ('routeros', 'RouterOS'),
+    ('esxi', 'ESXi'),
+    ('windows', 'Windows'),
+    ('ilo', 'iLO'),
+    ('other', 'Other'),
```

Comment:
Previously we had `None` type for hosts where OS is not applicable, like anycast services. How does that translate here?

Reply:
I've added a (None, 'None') OS choice, which should translate to NULL in SQL.